libdieharder/rng_file_input.c

```diff
@@ -325,12 +325,12 @@ static void file_input_set (void *vstate, unsigned long int s)
        }
      }
      if(strncmp(splitbuf[0],"count",5) == 0){
-       state->flen = atoi(splitbuf[1]);
+       state->flen = atoll(splitbuf[1]);
```

Comment:
This is the main change. `count` might be larger than `INT_MAX`, thus `atoll` is preferred.